toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -759,7 +764,7 @@ func shrinkFilesystemsHelper(buildImageFile string, verity []imagecustomizerapi.
 }
 
 func customizeVerityImageHelper(buildDir string, baseConfigPath string, config *imagecustomizerapi.Config,
-	buildImageFile string, partIdToPartUuid map[string]string,
+	buildImageFile string, partIdToPartUuid map[string]string, ukiEnabled bool,
```

Comment:
`ukiEnabled` can be inferred from `config`. So, it probably isn’t worth passing an extra param.